api/v1/bmc.proto

```diff
@@ -19,7 +18,7 @@ service BMC {
 message NetworkSourceRequest {
     v1.Authn authn = 1;
     v1.Vendor vendor = 2;
-    NetworkSource network_source = 3 [(validator.field) = {is_in_enum : true}];
```

Comment:
I'm not opposed to dropping these validations, I would just like to be sure that the validations are handled. I could be missing it, but i don't see them handled in any other way in the PR?

Reply:
I'm not sure how to support validations in the proto definitions without the library but with the library I can't generate client code for languages other than go. I'll do some more research to see what can be done for validations.
Do you happen to know what the original intent was behind adding the validations into the protobuf definitions?

Reply:
Well, I went back to this again on the ruby side and found the right way to build this while keeping the validations, I'll update the PR shortly to reflect this.